portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Dominic Terrones</title>
+    <link rel="icon" href="dt_wp.ico">
+    <meta name=description content="Gallery of Photos!">
+    <link rel="stylesheet" href="style.css">
+    <script src="slideshow.js"></script>
+  </head>
+  <div class="navbar">
+    <a id="box-1" href="index.html">Home</a>
+    <a id="box-2" href="recent.html">Recent Work</a>
+    <a id="box-3" href="gallery.html">Gallery</a>
+    <a id="box-4" href="contact.html">Contact</a>
+  </div>
+  <body>
+  <!-- Slideshow container -->
+  <div class="slideshow-container">
+
+    <!-- Full-width images with number and caption text -->
+    <div class="mySlides fade" id="first">
+      <div class="numbertext">1 / 3</div>
+      <img src="images/gallery/gsite_bg_17.jpeg" style="width:100%">
+      <div class="text">Hawaii!</div>
+    </div>
+
+    <div class="mySlides fade">
+      <div class="numbertext">2 / 3</div>
+      <img src="images/gallery/gsite_bg_2.jpg" style="width:100%">
+      <div class="text">Caption Two</div>
```

Comment:
no captions for image 2 & 3?  😆

Reply:
Agh! I would like you to believe that I visited a place called "Caption Two"...